executor/show_test.go

```diff
@@ -30,7 +31,17 @@ func (s *testSuite) TestShow(c *C) {
 		s.cleanEnv(c)
 		testleak.AfterTest(c)()
 	}()
+
 	tk := testkit.NewTestKit(c, s.store)
+
+	tk.Se = nil
```

Comment:
This explicitly force a new session to be created so that it won't break when someone else added some code before this piece of code later.